accounts-db/src/accounts_index.rs

```diff
@@ -1154,6 +1174,17 @@ impl<T: IndexValue, U: DiskIndexValue + From<T> + Into<T>> AccountsIndex<T, U> {
         self.get_and_then(pubkey, |entry| (false, entry.is_some()))
     }
 
+    /// Is `pubkey`, with `ancestors` and `max_root`, in the index?
+    pub fn contains_with(
```

Comment:
pub(crate)?

Reply:
Marking them #[allow(dead_coce)] sound good to me.

Reply:
> Done. Marked as dead code in [02b209e](https://github.com/solana-labs/solana/commit/02b209e599cca8d0b4645d5741eda7b7b9d03876).

I'm ok with marked as dead code. Isn't there a cfg[test] or something? Not a huge deal. Just trying to get it right with each commit.

Reply:
> #[cfg(test)]

yes, that's what I'd prefer.

Reply:
> I still think `pub` is the right visibility

I'm inclined to say make as little `pub` as possible. If this can be not pub, don't make it pub. Then we can get rid of it or change it later without any concern.